lib/tasks/database_cleanup.rake

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+namespace :database do
+  desc 'Check and remove orphaned objects'
+  task cleanup_orphans: :environment do
+    puts 'Checking and removing orphaned objects...'
+
+    # Tables to exclude from orphaned objects check
+    excluded_tables = ['cms_templates']
+
+    provider_account_ids = Account.where(provider: true).pluck(:id)
+
+    ActiveRecord::Base.descendants.each do |model|
```

Comment:
Better this IMO:

```suggestion
    ApplicationRecord.descendants.each do |model|
```

Reply:
Addressed in https://github.com/3scale/porta/pull/3732/commits/ee9e0a8b7a3dbe6c8ba742f6649b0618b1bc0f06